src/mode/modeHandler.ts

```diff
@@ -129,50 +129,63 @@ export class VimSettings {
   useCtrlKeys     = false;
 }
 
+export enum SearchDirection {
```

Comment:
I think it improves readability from the outside, more than having to determine the magic numbers 1 and -1. The particular values are only used internally.